python/lsst/meas/algorithms/normalizedCalibrationFlux.py

```diff
@@ -175,16 +177,30 @@ def run(self, *, exposure, catalog):
         if self.config.do_measure_ap_corr:
             ap_corr_field, ap_corr_err_field = self._measure_aperture_correction(exposure, catalog)
         else:
+            use_identity = False
             ap_corr_map = exposure.info.getApCorrMap()
             if ap_corr_map is None:
-                raise RuntimeError("NormalizedCalibrationFluxTask is configured with "
-                                   "do_measure_ap_corr=False but exposure has no aperture correction map.")
-            ap_corr_field = ap_corr_map.get(raw_flux_name)
-            ap_corr_err_field = ap_corr_map.get(raw_fluxerr_name)
-            if not ap_corr_field or not ap_corr_err_field:
-                raise RuntimeError("NormalizedCalibrationFlux is configured with "
-                                   f"do_measure_ap_corr=False but {raw_flux_name}/{raw_fluxerr_name} "
-                                   "not in exposure aperture correction map.")
+                self.log.warning(
+                    "Exposure does not have a valid normalization map for %r; using identity normalization.",
+                    exposure_id,
```

Comment:
I don't think you need to go out of your way to include the `exposure_id` in the log message because the logging system stores the dataId in every line.